huaweicloud-sdk-python-modelarts-dataset/modelarts/test/test_manifest_write_classification.py

```diff
@@ -37,26 +37,27 @@ def create_manifest():
         annotation_name = "Dog"
       annotation_creation_time = "2019-02-20 08:23:06"
       annotation_format = "manifest"
+      annotation_property = {"color": "black"}
       annotation_confidence = 0.8
       annotated_by = "human"
       annotations_list.append(
         Annotation(name=annotation_name, type=annotation_type,
                    confidence=annotation_confidence,
                    creation_time=annotation_creation_time,
-                   annotated_by=annotated_by, annotation_format=annotation_format))
+                   annotated_by=annotated_by, annotation_format=annotation_format, property=annotation_property))
     sample_list.append(
       Sample(source=source, usage=usage, annotations=annotations_list, inference_loc=inference_loc))
   return DataSet(sample=sample_list, size=size)
 
 
 def main(argv):
-  path = os.path.abspath('../') + "/resources/flowers-xy-V201902220937263726_2.manifest"
+  path = os.path.abspath('../../../') + "/resources/classification-xy-V201902220937263726_2.manifest"
```

Comment:
use os.sep to replace `\` to make the code can run on both linux and windows.

Reply:
Which system can't work? I test this test class in window 7, it can work successfully!